src/internal/types.ts

```diff
@@ -302,3 +303,73 @@ interface ReadableStreamDefaultReaderLike<T> {
 export interface ReadableStreamLike<T> {
   getReader(): ReadableStreamDefaultReaderLike<T>;
 }
+
+/**
+ * The global configuration object for RxJS, used to configure things
+ * like how to react on unhandled errors. Accessible via {@link config}
+ * object.
+ */
+export interface GlobalConfig {
```

Comment:
Can the interface declaration/docs be move back into the `src/internal/config.ts` file? And if it needs to be exported to be included in the docs, can the export be added here?

https://github.com/ReactiveX/rxjs/blob/c1c15db2448f8458de100a8d36c725c9c6bb0c14/src/index.ts#L86

Reply:
Yeah. It would be preferable if it were in the same file. It's not something that gets used all over the library like say... `ObservableInput` or something.

Reply:
Yes, this works. I'll move.